introduction_to_amazon_algorithms/jumpstart-foundation-models/text-generation-benchmarking/inference-benchmarking-custom-probe-example.ipynb

```diff
@@ -0,0 +1,147 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #9.            self.increment_value = 25
</pre><p>Isn't this choice depends heavily on the input/output size? For instance for inputs and outputs of the range of 2k/4k, this would result in max concurrent requests of 0/1. May be abstract it out?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/aws/amazon-sagemaker-examples/pull/4443/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='2'/>